app/views/welcome/_next-event.html.erb

```diff
@@ -10,7 +10,7 @@
 
       <ul>
         <% next_event.presentations.each do |presentation| %>
-          <li><%= "#{presentation.presenter} \u2013 #{presentation.topic}
+          <li><%= "#{presentation.presenter_name} \u2013 #{presentation.topic}
```

Comment:
So that it displays as an 'em dash' (—) vice an 'en dash' (–) or hypen  (-). The mock given for this story used an em dash, or at least what most resembled an em dash :)